cedar-lean/Cedar/Thm/Partial/Evaluation/Evaluate.lean

```diff
@@ -418,7 +418,6 @@ theorem subst_preserves_evaluation_to_value {expr : Expr} {req req' : Partial.Re
       have := List.sizeOf_lt_of_mem h₂
       exact subst_preserves_evaluation_to_value wf_r wf_e wf_s h_req hx
     exact Call.subst_preserves_evaluation_to_value ih h_req v h₁
-termination_by expr
```

Comment:
for this one, I think I remember leaving this one because if the proof does break, having the `termination_by` explicit led to a better error message than if you remove this line.  (Without this line, something like "failed to find a termination measure"; with this line, some actual failed proof obligation where the `expr` can't be proved to be smaller at a particular call site.)  That said, I'm fine if we want to adopt the convention to omit this line whenever Lean allows it.  It's easy enough to add it back (even locally) while troubleshooting termination proofs.

Reply:
Yeah, I'd buy that leaving them makes it easier to debug in the future if things break when heuristics change. Super satisfying to delete them though 😅 @emina any opinions?